09_Hyper-parameter_tuning_using_Ray/reference_solution/run.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#SBATCH --account=project_465001063
+#SBATCH --partition=standard-g
+#SBATCH --nodes=1
+#SBATCH --gpus-per-node=8
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=56
+#SBATCH --mem=0
+#SBATCH --time=0:15:00
+#SBATCH --output=output_ray.txt
+
+# Set up the software environment
+module purge
+module use /project/project_465001063/modules
```

Comment:
These have now been moved.
```suggestion
module use /appl/local/training/modules/AI-20240529
```